src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -152,18 +152,21 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
       return
     }
 
+    const artwork = this?.props?.order?.lineItems?.edges?.[0]?.node?.artwork
     const listPriceCents = this.props.order.totalListPriceCents
-    const artworkPrice = this?.props?.order?.lineItems?.edges?.[0]?.node
-      ?.artwork?.price
+    const artworkPrice = artwork?.price
+    const isInquiryCheckout = !artwork?.isPriceRange && !artwork?.price
```

Comment:
I don't want to touch on this subject here, it's not the scope of the ticket but I strongly believe we should not rely on their 2 properties to check if it's inquiry.

Reply:
Agree -- quite confusing

Reply:
We already discussed but adding here for visibility:
It's a bad flag name as it's not all-encompassing for inquiry checkout, but these conditions can only be met here if we're on an inquiry checkout scenario. We'll need to change the name of the flag, or maybe check if checking for a hidden price instead would accomplish the same (on another PR, of course).